src/clj/game/core/installing.clj

```diff
@@ -435,13 +435,17 @@
   (assert (or (event? card) (operation? card)) "condition counter must be event or operation")
   (let [cdef (card-def card)
         abilities (ability-init cdef)
+        corp-abilities (corp-ability-init cdef)
```

Comment:
this reminds me, `corp-abilities` and `runner-abilities` are dumb, we should just have a flag on abilities about which player can use them lmao

Reply:
Agreed, these `corp-abilities` and `runner-abilities` being separate has bit us quite a bit with some of the work we've done lately.

Reply:
I'll tackle it at some point, no need for it to be done here, just thinking out loud lol.